CMSgov/mint-app
# [EASI-3684] - Quality reporting frequency

# EASI-3684

Because this PR pulls into the git history of another ticket, use [this link](https://github.com/CMSgov/mint-app/compare/5bd0677..8ba6c0d?expand=1) to view the real diff

## Changes and Description

[Figma](https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?type=design&node-id=7150-111064&mode=design&t=jviG8roxKHYCbuwr-0)

BE Changes
- Added new field to schema - `qualityReportingFrequency`
- Added `Continually` and`Other` string fields
- Added migration to add column on `plan_ops_eval_and_learning`
- Added test for new field
- Updated postman collection

FE Changes
- Added new field `qualityReportingFrequency` to queries
- Implemented `FrequencyForm` for new question
- Added to Readonly and CSV
- Updated unit tests



## How to test this change

- Navigate to second to last page of opsEvalAndLearning
- Make selections on quality reporting frequency question
- Verify data persists on save
- Verify render on readonly
- Verify csv export data

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
